yaksh/views.py

```diff
@@ -954,3 +963,71 @@ def design_questionpaper(request):
         context = {'form': form, 'questionpaper':True}
         return my_render_to_response('yaksh/design_questionpaper.html',
                                      context, context_instance=ci)
+
+
+@login_required
+def download_questions(request):
+    user = request.user
+    if not is_moderator(user):
+        raise Http404('You are not allowed to view this page!')
+    questions = Question.objects.filter(user_id = user.id)
+    response = HttpResponse(content_type='text/csv')
+    response['Content-Disposition'] = 'attachment; filename="{0}_Questions.csv"'.format(user)
+    writer = csv.writer(response)
```

Comment:
I think it will be much easier if you write it as json and not csv.  CSV is very limiting.  It will also be much easier to just export/import json rather than csv.